spec.md

```diff
@@ -2450,6 +2450,26 @@ asClock(x)
 
 [@sec:primitive-operations] will describe the format and semantics of each primitive operation.
 
+## Primitive Property Operations {#sec:expressions:primitive-property-operations}
```

Comment:
It's kind of unfortunate that the PR has to go through this just because two different subsections have the same name. I should see if there is a pandoc option to do this automatically.

Reply:
Yeah, I just sort of followed how HW primitives do it--both naming the subsection the same as the section, and using these anchors to disambiguate. If there is a better option with pandoc, happy to use that.